res/css/views/rooms/_EventTile.scss

```diff
@@ -723,20 +723,14 @@ $threadInfoLineHeight: calc(2 * $font-12px); // See: _commons.scss
     flex-flow: wrap;
     align-items: center;
 
-    &:hover {
+    &:hover,
+    // Override .mx_EventTile:not([data-layout=bubble]).mx_EventTile:hover .mx_EventTile_line
+    &:not([data-layout=bubble]):hover .mx_EventTile_line {
         background-color: $system;
     }
 
-    // To cancel "&.mx_EventTile:hover .mx_EventTile_line"
-    &:not([data-layout=bubble]):hover {
-        .mx_EventTile_line {
-            background-color: $system;
-        }
-
-        // Override .mx_EventTile:not([data-layout=bubble]):hover.mx_EventTile_verified .mx_EventTile_line
-        &.mx_EventTile_verified .mx_EventTile_line {
-            box-shadow: none; // don't show the verification left stroke in the thread list (data-shape=ThreadsList)
-        }
+    &:not([data-layout=bubble]):hover .mx_EventTile_line {
+        box-shadow: none; // don't show the verification left stroke in the thread list
```

Comment:
This line removes the left stroke in the thread list:

https://user-images.githubusercontent.com/3362943/167669373-4896be99-9a02-4cc0-a841-34c11f266483.mp4